### PR TITLE
Feat/1496 remove dead code

### PR DESCRIPTION
**Pull request by ShynRou**
Part of #1496.

Removed old unused and out commented code. If you notice any components that were intended for future usage, please note them here.

### CODE REVIEW

**Review by creed-victor — COMMENTED** [review]
Everything from `components/Form` should be used i think - these are components which should have used in entire dapp

File: src/app/pages/MarginTradePage/components/TradeDialog/index.tsx [hunk 1]
```diff
@@ -155,26 +153,6 @@ export function TradeDialog() {
...
-          {/*  <Slider*/}
-          {/*    value={slippage}*/}
-          {/*    onChange={e => setSlippage(e)}*/}
-          {/*    min={0.1}*/}
-          {/*    max={1}*/}
-          {/*    stepSize={0.05}*/}
-          {/*    labelRenderer={value => <>{value}%</>}*/}
-          {/*    labelValues={[0.1, 0.25, 0.5, 0.75, 1]}*/}
-          {/*  />*/}
-          {/*</FormGroup>*/}
```
Comment (creed-victor) [inline src/app/pages/MarginTradePage/components/TradeDialog/index.tsx:176]: Will use this soon

**Review by creed-victor — COMMENTED** [review]
I think /pages/OriginsLaunchpad files will be used, just not used right now because there is no active sale on it. @tiltom

**Comment by ShynRou** [comment]
@creed-victor re-added requested files, and out commented code. Added a TODO comment on the TradeDialog to indicate it's future usage.

**Comment by tiltom** [comment]
> I think /pages/OriginsLaunchpad files will be used, just not used right now because there is no active sale on it. @tiltom

You're right Victor, `pages/OriginsLaunchpad` will be used but there is no sale yet.

**Comment by ShynRou** [comment]
Just for clarity: components/Form/* and pages/OriginLaunchpad are already reinstated, and this PR can be reviewed.

**Review by creed-victor — DISMISSED** [review]

**Review by soulBit — APPROVED** [review]

**Pull request merged**